src/Pos/Explorer/Web/Transform.hs

```diff
@@ -64,7 +69,7 @@ explorerServeWebReal sendActions = explorerServeImpl . explorerApp $
 
 nat :: ExplorerProd (ExplorerProd :~> Handler)
 nat = do
-    tlw        <- askTxpMem
+    tlw        <- Ether.ask @TxpHolderTag -- askTxpMemAndMetrics -- askTxpMem
```

Comment:
These comments are left here as a reminder. 
If we want to use functions from `Pos.Txp`, we need to have `askTxpMemAndMetrics` exported first.